doozerlib/olm/bundle.py

```diff
@@ -24,6 +24,25 @@ class OLMBundle(object):
     def __init__(self, runtime):
         self.runtime = runtime
 
+    def find_bundle_for(self, operator_nvr):
+        """Check if a bundle already exists for a given `operator_nvr`.
+
+        :param string operator_nvr: Operator NVR (format: my-operator-v4.2.30-202004200449)
+        :return string: NVR of latest found bundle build, or None.
+        """
+        self.operator_nvr = operator_nvr
+        self.get_operator_buildinfo()
+
+        builds = brew.get_tagged_builds(tags=[self.target],
+                                        build_type='image',
+                                        event=None,
+                                        session=self.brew_session)[0]
```

Comment:
this gets all the images in the tag, which still seems excessive... and I forgot `brew.get_tagged_builds` really doesn't do what I expected here. This really should be simply
```suggestion
        builds = self.brew_session.listTagged(tag=self.target, package=self.bundle_brew_component)
```